buddyboss/buddyboss-platform
# Messages - improve the avatar display for multi-user message threads

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1050

### How to test the changes in this Pull Request:

1. I should never see my own avatar in a conversation. Should always be the most recent user(s) to reply who are not me.
2. If a conversation has 3 or more people (including me) then we see double avatar layout, with the two most recent replies (not me) showing as avatars. Similar to iMessage - https://prnt.sc/t1txa9
3. Apply these changes into message dropdown in title bar also
4. Update API so the app gets correct data to be able to replicate this UI.

### Proof Screenshots or Video
Video explanation - https://www.loom.com/share/912bc86bfd064f9790f4627c0f56a689



### Other information:




Theme PR: https://github.com/buddyboss/buddyboss-theme/pull/829
